packages/help-reference/src/DefinedMedia.tsx

```diff
@@ -0,0 +1,160 @@
+import { PropsWithChildren } from 'react';
+import {
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableRow,
+	TableWrapper,
+} from '@ag.ds-next/react/table';
+import { AlertFilledIcon, SuccessFilledIcon } from '@ag.ds-next/react/icon';
+import { Flex } from '@ag.ds-next/react/flex';
+
+const SuccessCell = () => (
+	<TableCell verticalAlign="middle">
+		<Flex>
+			<SuccessFilledIcon color="success" />
+			&nbsp;&nbsp;Yes
+		</Flex>
+	</TableCell>
+);
+
+const ErrorCell = () => (
+	<TableCell verticalAlign="middle">
+		<Flex>
+			<AlertFilledIcon color="error" />
+			&nbsp;&nbsp;No
+		</Flex>
+	</TableCell>
+);
+
+const PsuedoHeaderCell = (t: PropsWithChildren) => (
+	<TableCell>{t.children}</TableCell>
+);
+
+export const definedMedia = {
```

Comment:
I feel strange about bundling this into the component. Seems less-worse than the alternative however.